HW3.Rmd

```diff
@@ -0,0 +1,22 @@
+---
+title: "HW3"
+author: "Sherzod Tokamov"
+date: "7/20/2020"
+output:
+  pdf_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(ggplot2)
+angles <- read_csv("angles.csv", stringsAsFactors = FALSE) #read my csv file
```

Comment:
Giving me an error when using stringAsFactors=FALSE and I'm unsure why since this is how it was done in the datacamp exercises. Maybe I am missing a package or something?